frontend/src/open-api/api/default-api.ts

```diff
@@ -296,6 +387,48 @@ export const DefaultApiAxiosParamCreator = function (
 export const DefaultApiFp = function (configuration?: Configuration) {
   const localVarAxiosParamCreator = DefaultApiAxiosParamCreator(configuration);
   return {
+    /**
+     *
+     * @param {*} [options] Override http request option.
+     * @throws {RequiredError}
+     */
+    async authorizationTest(
+      options?: any
+    ): Promise<
+      (axios?: AxiosInstance, basePath?: string) => AxiosPromise<string>
+    > {
+      const localVarAxiosArgs =
+        await localVarAxiosParamCreator.authorizationTest(options);
+      return createRequestFunction(
+        localVarAxiosArgs,
+        globalAxios,
+        BASE_PATH,
+        configuration
+      );
+    },
+    /**
+     *
+     * @param {ChangePasswordDTO} changePasswordDTO
+     * @param {*} [options] Override http request option.
+     * @throws {RequiredError}
+     */
+    async changePassword(
```

Comment:
@frontend devs, functions in this file are the ones you should be using when calling the backend api. no more need to use axios directly :)

so for example, if you want to make a request to `/api/v1/auth/change-password`, you would just use this function. it can be directly imported from `generated-api`